scratch/youLike.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
+
+import os
+import pickle
+import json
+from urllib.request import urlopen
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+from google_auth_oauthlib.flow import InstalledAppFlow
+
+# scrape youtube video ids from the channel id specified in the url
+url = 'https://www.googleapis.com/youtube/v3/search?key={your_api_key}&channelId=UCLgQwtSa4EsuJge-N8Bjs3g&part=snippet,id&order=date&maxResults=20'
```

Comment:
IT's usually good practice to just keep your global code in one place. For example, you have a __main__ section at the bottom of the script and this kind of global code would be better located there.